api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java

```diff
@@ -97,6 +106,13 @@
     @Inject
     private WhitelistEntryDataService whitelistEntryDataService;
 
+    @Inject
+    private CallDetailRecordDataService callDetailRecordDataService;
+
+    public UserControllerBundleIT() {
+        System.setProperty("org.motechproject.testing.osgi.http.numTries", "1");
```

Comment:
Not if it's only going to try once only.
The wait happens after a failure, and not for the last failure, so since
there's only one try it'll never wait.
But FYI the default values are still used if nothing is specified.

On Fri, Apr 24, 2015 at 2:41 PM, Rob LaRubbio <notifications@github.com>
wrote:

> In
> api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java
> <https://github.com/koshalt/mim/pull/90#discussion_r29088254>:
>
> > @@ -97,6 +106,13 @@
> >      @Inject
> >      private WhitelistEntryDataService whitelistEntryDataService;
> >
> > +    @Inject
> > +    private CallDetailRecordDataService callDetailRecordDataService;
> > +
> > +    public UserControllerBundleIT() {
> > +        System.setProperty("org.motechproject.testing.osgi.http.numTries", "1");
>
> Should this also have the wait time property?
>
> System.setProperty("org.motechproject.testing.osgi.http.msWait", "1000");
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/90/files#r29088254>.
>



-- 
Frank Huster
Mobile Health Innovations
Grameen Foundation
+1.206.325.6690 x201